llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -209,6 +209,12 @@ cl::opt<bool> DeviceGlobals{
     cl::desc("Lower and generate information about device global variables"),
     cl::cat(PostLinkCat)};
 
+cl::opt<bool> GenerateDeviceImageWithDefaultSpecConsts{
```

Comment:
I've just realized that this option is likely incompatible with `--ir-output-only`, let's add a corresponding check to `main`.